apiserver/facades/client/highavailability/highavailability_test.go

```diff
@@ -621,14 +625,14 @@ func (s *clientSuite) TestEnableHABootstrap(c *gc.C) {
 }
 
 func (s *clientSuite) TestHighAvailabilityCAASFails(c *gc.C) {
-	f, release := s.NewFactory(c, s.ControllerModelUUID())
-	defer release()
+	c.Skip("TODO - reimplement when facade moved off of mongo")
+	ctx := context.Background()
 
-	st := f.MakeCAASModel(c, nil)
-	defer st.Close()
+	mockModelInfo := highavailability.NewMockModelInfoService(gomock.NewController(c))
```

Comment:
Don't bother with these changes since the test is skipped. Just leave the original code in place

Reply:
fixed. but added context.Background() to keep the code function call correct.